### DISCUSSION TITLE
Is there any option to add a YouTube video?

### DISCUSSION THREAD

**Discussion by munimrahman** [igordanchenko/yet-another-react-lightbox#252] [Q&A] [answered]
I want to use a YouTube embedded video link, but I cannot find any option to add a YouTube video.

**Comment by igordanchenko** [comment, marked as answer]
There are multiple working examples that you could find in previous discussions and issues. Here is one working example for your reference - https://github.com/igordanchenko/yet-another-react-lightbox/issues/49#issuecomment-1505689829

And here is a live demo with a custom plugin - https://stackblitz.com/edit/yet-another-react-lightbox-youtube-example?file=src%2FYouTube.tsx
